web/tests/help-page.spec.ts

```diff
@@ -0,0 +1,62 @@
+//Tests the help page
+import { test, expect } from '@playwright/test';
+import { TableBrowser, Workspace } from './lynxkite';
+
+let workspace: Workspace;
+
+async function checkExactlyOneCurrent() {
+    // There is only one highlighted item:
+    await expect(await workspace.page.locator('span.find-highlight-current').count()).toBe(1);
+}
+
+async function checkCurrentHighlight(expectedPos) {
+    await checkExactlyOneCurrent();
+    // Check correct item:
+    const cssQuery = 'span.find-highlight.find-highlight-current.find-highlight-' + expectedPos;
+    await expect(workspace.page.locator(cssQuery)).toBeVisible();
+}
+
+test.beforeAll(async ({ browser }) => {
+    workspace = await Workspace.empty(browser);
+});
```

Comment:
You don't need a workspace here, right? Delete this.

Reply:
I think I need it to access the Page interface later. Do you suggest using the Splash class for it?
await workspace.page.goto('#/help'); 
vs 
await splash.page.goto('#/help');

Reply:
> I think I need it to access the Page interface later.

Of course. Just use the page you get from Playwright. Please look at the docs at some point. https://playwright.dev/docs/writing-tests#first-test